src/storage/src/client/controller.rs

```diff
@@ -69,6 +70,10 @@ include!(concat!(env!("OUT_DIR"), "/mz_storage.client.controller.rs"));
 static METADATA_COLLECTION: TypedCollection<GlobalId, CollectionMetadata> =
     TypedCollection::new("storage-collection-metadata");
 
+// The UUID here is arbitrary
+pub static STATUS_SHARD_ID: Lazy<ShardId> =
```

Comment:
I'm not sure we'd want to have this arbitrary  shard id here? We could make `status_shard` an optional field to get around having to come up with a dummy one?

Reply:
It's mostly a way to ensure for now that all sources use the same `ShardId` without having to create an arbitrary one at runtime and persisting it somewhere (probably in Adapter?). Even if it becomes optional, we'd still have to have a hardcoded value somewhere for now to ensure that all sources use the same shard.

An alternative would be to hardcode it in Adapter (for now), and pass it as an extra `Option<ShardId>` argument in `CollectionDescription`. Would you prefer that instead?